apps/portals/run.sh

```diff
@@ -77,7 +77,7 @@ EOL
 
 elif [ "$1" = "push-staging" ]; then
   # sync current with staging
-  pnpm i && pnpm build
+  pnpm i && pnpm nx run portals:build
```

Comment:
Using Nx ensures that all of the local dependencies of portals will also be built.